packages/sdk-starter-kit/src/types.ts

```diff
@@ -46,22 +46,23 @@ export type SafeConfig = {
   saltNonce?: string
 }
 
-export type ExistingSafeKitConfig = {
+export type ExistingSdkStarterKitConfig = {
```

Comment:
Thing is that, now that its renamed, this name became really "avant-garde".

Reply:
Agree, I changed it